pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -154,3 +157,9 @@ type Config struct {
 	// +optional
 	PriorityClassName string `json:"priorityClassName,omitempty"`
 }
+
+type Platforms struct {
+	// OpenShift allows configuring openshift specific components and configurations
+	// +optional
+	OpenShift OpenShift `json:"openshift,omitempty"`
```

Comment:
Can we add a doc as its change in configuration behavior 

Or will it be handled in downstream because its part of Openshift ?

Reply:
yes will be in downstream
we can upstream too but got some other things to do for 1.9 😅 so will do later

Reply:
Sure can you create an issue for reference and do it later

Reply:
https://github.com/tektoncd/operator/issues/1138